.github/workflows/tests.yml

```diff
@@ -19,3 +19,12 @@ jobs:
         with:
           secret: ${{ secrets.FIREBASE_GOOGLE_SERVICE }}
       - run: ./gradlew ${{ matrix.task }}
+  ios:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 17
+      - run: ./gradlew shared:iosX64Test
```

Comment:
Failing test `IosGreetingTest` results in failing the job so the initial setup seems to be enough